package/config_types/config_list.js

```diff
@@ -0,0 +1,79 @@
+/**
+  * @class
+  * @extends { Array }
+*/
+class ConfigList extends Array {
+  get(key) {
+    const index = this.getIndex(key, true)
+    return this[index].value
+  }
+
+  set(key, value) {
+    return this.add({ key, value })
+  }
+
+  append(key, value) {
+    return this.set(key, value)
```

Comment:
This should call `add` directly instead of `set`.

Reply:
👍